moped-database/README_seed_data.md

```diff
@@ -36,10 +41,17 @@ $ docker run -it --rm --network host \
     postgres://moped:moped@localhost:5432/moped > seeds/1602292389297_initial_seed_staging.sql
 ```
 
-6. At the top of the updated seed file (), check make sure the search path configuration looks like this - change it if necessary:
+6. You will need to make two changes to the updated seed field. At the top of the seed file, the first line may have a warning message that you need to delete:
+
+```
+pg_dump: warning: there are circular foreign-key constraints on this table
+```
```

Comment:
i'm not quite sure why the circular key warning is popping up after adding these tables. the data will import without issue 🙈